mLRS/Common/hal/esp/tx-hal-radiomaster-bandit-micro-900-esp32.h

```diff
@@ -65,8 +67,14 @@
 //#define DEVICE_HAS_NO_COM
 #define DEVICE_HAS_NO_DEBUG
 
-#define DEVICE_HAS_SINGLE_LED
-#define DEVICE_HAS_I2C_DISPLAY_ROT180
+#ifdef TX_ELRS_RADIOMASTER_BANDIT_900_ESP32
+    #define DEVICE_HAS_SINGLE_LED_RGB
```

Comment:
pl only two blanks for idented defines
also no is ok

Reply:
Removed idents